src/add_and_remove.js

```diff
@@ -0,0 +1,59 @@
+function addNewTask() {
+  const taskArray = JSON.parse(localStorage.getItem('tasks'));
+  const description = document.getElementById('task-description').value;
+  taskArray.push({
+    description,
+    completed: false,
+    index: taskArray.length + 1,
+  });
+  document.getElementById('task-description').value = '';
+  localStorage.setItem('tasks', JSON.stringify(taskArray));
+}
+
+function deleteTask(task) {
+  const taskArray = JSON.parse(localStorage.getItem('tasks'));
+  const sortedTasks = taskArray.sort((a, b) => a.index - b.index);
+  sortedTasks.splice(task.index - 1, 1);
+  sortedTasks.forEach((task, index) => {
+    task.index = index + 1;
+  });
+  localStorage.setItem('tasks', JSON.stringify(sortedTasks));
+}
+
+function editTask(e, listItem, textElement, spanItem, allTasks, task, populate) {
+  const menu = e.target;
+  menu.classList.remove('fa-ellipsis-v');
+  menu.classList.add('fa-trash-alt');
+  listItem.classList.add('bg-warning');
+  textElement.remove();
+  const inputElement = document.createElement('input');
+  inputElement.value = task.description;
+  inputElement.classList.add('no-focus', 'border-0', 'bg-warning');
+  spanItem.append(inputElement);
```

Comment:
Creating elements like this can sometimes be cumbersome and also leads to having many lines of code in your modules. Consider refactoring the creation of an element together with all its attributes to one helper method. For example 👇🏼:

###  createElementHelper.js
```JavaScript
const createElement = (tag, attributes = {}, textContent = '') => {
  const element = document.createElement(tag);
  const keys = Object.keys(attributes);
  keys.forEach((key) => {
    element.setAttribute(key, attributes[key]);
  });
  element.textContent = textContent;
  return element;
};

module.exports = { createElement };
```

### How to apply
```JavaScript
import { createElement  } from './createElementHelper.js';

const paragraph = createElement('p', { class: 'paragraph' }, "Some paragraph");
```

From the above example, we can now realize that the creation of a `<p>` element has just taken one line of code.

Reply:
Thank you for the review